find-address/src/main/java/com/esri/arcgisruntime/sample/findaddress/MainActivity.java

```diff
@@ -0,0 +1,309 @@
+import android.provider.BaseColumns;
+import android.support.v4.content.ContextCompat;
+import android.support.v4.widget.SimpleCursorAdapter;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.SearchView;
+import android.util.Log;
+import android.view.MotionEvent;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.Callout;
+import com.esri.arcgisruntime.mapping.view.DefaultMapViewOnTouchListener;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.IdentifyGraphicsOverlayResult;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.mapping.view.WrapAroundMode;
+import com.esri.arcgisruntime.symbology.PictureMarkerSymbol;
+import com.esri.arcgisruntime.tasks.geocode.GeocodeParameters;
+import com.esri.arcgisruntime.tasks.geocode.GeocodeResult;
+import com.esri.arcgisruntime.tasks.geocode.LocatorTask;
+import com.esri.arcgisruntime.tasks.geocode.SuggestResult;
+
+public class MainActivity extends AppCompatActivity {
+
+  private final String TAG = MainActivity.class.getSimpleName();
+  private final String COLUMN_NAME_ADDRESS = "address";
+  private final String[] mColumnNames = { BaseColumns._ID, COLUMN_NAME_ADDRESS };
+  private SearchView mAddressSearchView;
+
+  private MapView mMapView;
+  private LocatorTask mLocatorTask;
+  private GraphicsOverlay mGraphicsOverlay;
+  private GeocodeParameters mAddressGeocodeParameters;
+  private PictureMarkerSymbol mPinSourceSymbol;
+  private Callout mCallout;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+
+    // inflate address search view
+    mAddressSearchView = (SearchView) findViewById(R.id.addressSearchView);
+    mAddressSearchView.setIconified(false);
+    mAddressSearchView.setFocusable(false);
+    mAddressSearchView.setQueryHint(getResources().getString(R.string.address_search_hint));
+
+    // define pin drawable
+    BitmapDrawable pinDrawable = (BitmapDrawable) ContextCompat.getDrawable(this, R.drawable.pin);
+    try {
+      mPinSourceSymbol = PictureMarkerSymbol.createAsync(pinDrawable).get();
+    } catch (InterruptedException | ExecutionException e) {
+      Log.e(TAG, "Picture Marker Symbol error: " + e.getMessage());
+      Toast.makeText(getApplicationContext(), "Failed to load pin drawable.", Toast.LENGTH_LONG).show();
+    }
+    // set pin to half of native size
+    mPinSourceSymbol.setWidth(19f);
+    mPinSourceSymbol.setHeight(72f);
```

Comment:
Set an offset on the marker symbol such that the `Callout` isn't hiding the marker, `mPinSourceSymbol.setLeaderOffsetY(25f);`

![offset-symbol-2](https://user-images.githubusercontent.com/1308001/30093733-47eeef6e-927c-11e7-8b75-620911d73d1e.png)

Reply:
I noticed the other samples did it, I just thought it looked a bit off since the leader drifts off the point on zoom. But happy to offset to keep in line with the other samples.

Reply:
added a MapScaleChangedListener to update leader position

Reply:
@TADraeseke Good catch, provided a fix for that in latest review.

Reply:
Just noticed that attaching the `GeoElement` does not resolve the leader drift issue without adding a `MapScaleChangedListener` so let's keep it in for now but use the `Callout.setGeoElement` approach.